charts/argo-events/values.yaml

```diff
@@ -66,6 +66,9 @@ configs:
       - version: latest
         natsStreamingImage: nats-streaming:latest
         metricsExporterImage: natsio/prometheus-nats-exporter:latest
+      - version: 0.22.1
```

Comment:
One more ask - can you add following `@default` annotation to not pollute README.md with all NATs versions?

```yaml
configs:
  ## NATS event bus
  nats:
    # -- Supported versions of NATS event bus
    # @default -- See [values.yaml]
    versions:
```

Reply:
@pdrastil Sure, no problem. But just so I'm clear, did you want me to revert the changes in `README.md` as well? Is the `README.md` something that's auto-generated by the doc gen routine?

Reply:
@rakhbari Yes you need to regenerate README.md via `scripts/helm-docs.sh` and commit that one as well

Reply:
@pdrastil 👍 done.